src/components/input/select/index.tsx

```diff
@@ -253,7 +270,14 @@ export class SmoothlyInputSelect implements Input, Editable, Clearable, Componen
 					<smoothly-icon size="tiny" name={this.open ? "caret-down-outline" : "caret-forward-outline"} />
 				</div>
 				<slot name="label" />
-				{this.open && <section onClick={() => (this.open = true)} />}
+				{this.open && (
+					<section
```

Comment:
Since we are in here. can we not use a section element. This is not the [use case for a section tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section#usage_notes).

Also a note for something future. We do not need to cover the screen with a an element to make something like this. We can use `@Listen("click", { target: "window" })` and use the event method `event.composedPath()` and see if it `.includes(...)` the options div element. 
If we cover the screen like we do here i think it should "gray out" the background.